pkgs/data/icons/buuf-icon-theme/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenvNoCC, fetchurl }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "buuf-icon-theme";
+  version = "3.45";
+
+  src = fetchurl {
+    url = "http://buuficontheme.free.fr/buuf${version}.tar.xz";
+    sha256 = "sha256-w6ertMv3ERkj9brJsxodo5JbaPeS0a29SOVkUK+o7MU=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/icons/
+    cp -r . $out/share/icons/${pname}
+  '';
+
+  meta = with lib; {
+    description = "Buuf icon theme";
```

This description doesn't say much more than the package name. How about something like this?

```suggestion
    description = "A comprehensive icon theme adapted for Gnome";
```